cpp/src/interop/from_arrow.cu

```diff
@@ -146,6 +152,59 @@ struct dispatch_to_cudf_column {
 
     return col;
   }
+
+  struct every_other {
+    __device__ size_type operator()(size_type i) { return 2 * i; }
+  };
+
+  template <typename T, typename std::enable_if_t<cudf::is_fixed_point<T>()>* = nullptr>
+  std::unique_ptr<column> operator()(arrow::Array const& array,
+                                     data_type type,
+                                     bool skip_mask,
+                                     rmm::cuda_stream_view stream,
+                                     rmm::mr::device_memory_resource* mr)
+  {
+    using DeviceType = device_storage_type_t<T>;
+
+    size_type const BIT_WIDTH_RATIO = 2;  // Array::Type:type::DECIMAL (128) / int64_t
```

Comment:
The comment is helpful, but can `2` be replaced with the equivalent equation? Then there wouldn't be a need for the comment.

Reply:
We tried digging around the Arrow API to get the width of the `Arrow::Type:type` width, but gave up trying to find it.

Reply:
Looks like `arrow::Decimal128::bit_width` should give you what you want:

https://github.com/apache/arrow/blob/bdbd1b361f903750918542093545c506de585049/cpp/src/arrow/util/basic_decimal.h#L45

It's defined in the `BasicDecimal128` type from which `Decimal128` inherits:

https://github.com/apache/arrow/blob/bdbd1b361f903750918542093545c506de585049/cpp/src/arrow/util/decimal.h#L50

Reply:
It looks like this was only added [4 months ago](https://github.com/apache/arrow/commit/feabdf071abac12e6dc796c115d1f7567bcac9e2) and it we don't have it in our version of Arrow.

Reply:
I suspect something similar was present in older versions of Arrow, but I think we're upgrading soon anyways, so maybe it doesn't matter.